spring-cloud-stream-binder-kafka/src/test/resources/logback.xml

```diff
@@ -4,11 +4,12 @@
             <pattern>%d{ISO8601} %5p %t %c{2}:%L - %m%n</pattern>
         </encoder>
     </appender>
+    <logger name="org.apache.kafka" level="DEBUG"/>
```

Comment:
Do we really need such a noise in logs?

Reply:
Those logs are harmless and seem to be useful when debugging tests.

Reply:
Right, that's in other projects we have such an entry with the `WARN` and let end-developer to override it for `DEBUG` in case of troubles.
But `DEBUG` is indeed harmful. Tests without it much faster.
But again: that is just my paranoia to pursue the short time for tests...

Reply:
Feel free to change to WARN on merge then.